### PR TITLE
[ar] Add Arabic technique glossary

### PR DESCRIPTION
**Pull request by mboukhalfa**
Adding Arabic technique glossary to unify and avoid using different translation for the same English word.
Unified Kubernetes translation based on the discussion : https://github.com/kubernetes/website/issues/45078

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/language ar

File: content/ar/docs/contribute/localization.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+
+<!-- body -->
+
+### معلومات المعجم التقني العربي
+
+المعجم التقني العربي يهدف لتوحد الفردات التقنية المعتمدة في وثائق كوبيرنيتيس العربية و تجنب إستعمال ترجمات مختلفة لنفس المصطلح الإنجليزي.
+تناقش المصطلحات المدرجة في المعجم التقني من خلال اجتماع لفريق الترجمة العربية أو من خلال إضافة قضية على جيتهاب.
+لايجب إستخدام أي مرادف بختلف عن المصطلح المدرج في المعجم في وثائق كوبيرنيتيس بل يجب تحديث المعجم أولا.
```
Comment (adowair) [inline content/ar/docs/contribute/localization.md:19]:
```suggestion
إذا وجد تعريب لمصطلح إنجليزي معين في المعجم، فيجب استخدام ذلك التعريب فقط، ولا يجوز استخدام أي تعريب أو ترجمة أخرى للمصطلح في وثائق كوبيرنيتيس. نرحب دائماً بطلبات تحديث المعجم ومناقشة التعريبات.
```

**Review by adowair — CHANGES_REQUESTED** [review]
Thank you for taking the initiative. I've made some additions to text and some typing nits.

File: content/ar/docs/contribute/localization.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: دليل لترجمة وثائق كوبيرنيتيس
```
[the remark this answers is not in the document]
Comment (mboukhalfa) [reply content/ar/docs/contribute/localization.md]: done

File: content/ar/docs/contribute/localization.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+content_type: concept
+---
+
+<!-- overview -->
+
+دليل لترجمة وثائق كوبيرنيتيس إلى اللغة العربية
```
[the remark this answers is not in the document]
Comment (mboukhalfa) [reply content/ar/docs/contribute/localization.md]: done

File: content/ar/docs/contribute/localization.md (hunk 1 above)
Comment (mboukhalfa) [reply content/ar/docs/contribute/localization.md]: done

**Comment by adowair** [comment]
LGTM for Arabic, thank you @mboukhalfa

**Review by adowair — APPROVED** [review]

**Comment by adowair** [comment]
Thanks @mboukhalfa. It may be more convenient and better Git hygiene to move updates to the existing docs to a new PR.

**Comment by ContainerNerd** [comment]
LGTM for Arabic!
Thank you fro the efforts @mboukhalfa

**Review by ContainerNerd — APPROVED** [review]

**Comment by seifrajhi** [comment]
LGTM for Arabic!
Thank you fro the efforts @mboukhalfa

**Comment by adowair** [comment]
> The idea is to unify the words once this PR gets merged so we do not have merged content conflicting with the technical glossary

@mboukhalfa I understand your sentiment to make sure the development branch is "clean" in this way, but I would like to respectfully rebut, for several reasons:
1. `dev-1.29-ar.1` is a development, unpublished branch. Since its content is not visible publicly, we can afford to have merged work in progress.
2. As more and more documents get added, it will become impractical to combine all updates to terminology localization in this one PR. What I fear is that way, this PR will not merge **until after** everything else is done, which I do not think is the intuitive order of operations.
3. Repeating the above, I fear that we regress into a "waterfall" model where all PRs are "pending" because we are waiting for "perfectly consistent" PRs. This will slow down progress and defeats the purpose of having a development branch and a Git history.
4. On that topic, as an aside, I believe that this project is set up with the ability to "squash" commits when PRs are merged, so I believe that we can indeed create incremental commits within PRs, saving the "history" of the work. As @seifrajhi pointed out in #45047, in reference to #40184, force-pushes break Netlify's rendering behavior.

The content of this PR (minus the last set of changes) looks great to me, and I really appreciate your effort. I am encouraging to merge it ASAP so that we can see real, even if incremental, progress.

**Comment by adowair** [comment]
Also, for reference, this PR pours into the effort: #44682.

**Comment by adowair** [comment]
/assign @seifrajhi

Will you please take a look and lgtm?

**Pull request merged**